### ISSUE TITLE
train model on own data?

### ISSUE THREAD

**Issue by xiaoxiao079** [Smith42/astroddpm#2]
hi there i want to train you model on our images. could it possible to train you model on grey scale images, and what are the changes which would needed to be done before training it on custom data?

**Comment by Smith42** [comment]
Yes you can train on greyscale images, you would need to normalise them with a new dataloader so that they have a scale between -1 and 1. You would also need to change the network input channels to be 1.
